### DISCUSSION TITLE
can I add a shared folder in network location as an additional working directory?

### DISCUSSION THREAD

**Discussion by Wufoofoo** [neurodesk/neurodesk.github.io-hugo#364] [Q&A] [answered]
Hi there, I tried to set the shared folder as an additional working directory but nothing is happening. Does that mean I am unable to do so? If that is the case, is there anything else I can do? the path is like \\shared.sydney.xxxx

**Comment by stebo85** [comment, marked as answer]
Dear @Wufoofoo,

Network locations as additional working directories in Windows are tricky and I haven't seen this working yet. You might be better off copying your data to a local directory and then using this as a working directory. Alternatively, you could ask if your IT can provide you with a Neurodesk instance where your data is mounted - this works very nicely in Linux, but is a bit more setup work.

Thank you
Steffen
